Azure/sonic-buildimage
# [mellanox]: Integrate CRIU tool to SYNCD docker container

Signed-off-by: Volodymyr Samotiy <volodymyrs@mellanox.com>



**- What I did**
Integrated CRIU tool to SYNCD docker container (it is needed for SYNCD warm-reboot support on Mellanox platforms)
**- How I did it**
Integrated to SONiC build system in order to build DEB package and then install it to SYNCD docker container
**- How to verify it**
Go to SYNCD docker and run ```criu``` tool:
```
root@sonic:/home/admin# docker exec syncd criu check
Looks good.
root@sonic:/home/admin#
```
**- Description for the changelog**
[mellanox]: Integrate CRIU tool to SYNCD docker container
